apps/api/src/services/ai-data-service.ts

```
@@ -1,733 +1,434 @@
-      // Apply clinic filter if provided
-      if (context?.domain) {
-        query = query.eq('clinic_id', context.domain);
+    let dateFilter = '';
+    if (financial?.period) {
+      const now = new Date();
+      switch (financial.period) {
+        case 'today':
+          dateFilter = `AND date::date = '${now.toISOString().split('T')[0]}'`;
+          break;
+        case 'week':
+          const weekStart = new Date(now);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)